app/models/content_item.rb

```diff
@@ -46,7 +46,7 @@ def self.create_or_replace(base_path, attributes)
   field :_id, :as => :base_path, :type => String, :overwrite => true
   field :content_id, :type => String
   field :title, :type => String
-  field :description, :type => String
+  field :description, :type => Hash, :default => { "value" => nil }
```

Comment:
It looks like having a `String` type field sets defaults the value to `nil`:

```ruby
ContentItem.create.title
#=> nil
```

So this should be ok?